src/brandbar.module.css

```diff
@@ -0,0 +1,13 @@
+.dropdownItem {
+  padding: 8px;
```

Comment:
This padding isn't wanted.

The landing page `dropdown-item`s incorrectly have the `nav-link nav-menu-button` classes.  The `nav-link` class adds some unwanted `padding: 0.5rem`.  We want the default padding for `.dropdown-item`.  (Or something similar to that.  Either way we should be using the default padding for `.dropdown-item`).

The fix to this is

1. Remove `nav-link nav-menu-button` from `dropdown-item` in flight landing page repo.
2. Remove this padding here.